doc/user/content/connect-sources/postgres-amazon-rds.md

```diff
@@ -0,0 +1,395 @@
+
+Materialize uses PostgreSQL's [logical replication](https://www.postgresql.org/docs/10/logical-replication.html) protocol to track changes in your datatabase and propogate them to Materialize.
+
+As a first step, you need to make sure logical replication is enabled.
+
+1. As a user with the role `rds_superuser`, connect `psql` to your database and check if logical replication is enabled:
+
+    ``` sql
+    SELECT name,setting
+      FROM pg_settings
+      WHERE name = 'rds.logical_replication';
+    ```
+
+    If logical replication is already on, skip to [Step 2](#step-2-create-a-publication):
+
+    ``` noftmt
+            name             | setting
+    -------------------------+---------
+    rds.logical_replication  | on
+    (1 row)
+    ```
+
+    If logical replication is off, continue to the next step:
+
+    ``` noftm
+            name             | setting
+    -------------------------+---------
+    rds.logical_replication  | off
+    (1 row)
+    ```
+
+1. [Create a custom RDS parameter group](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_WorkingWithDBInstanceParamGroups.html#USER_WorkingWithParamGroups.Creating).
+
+    Be sure to set **Parameter group family** to your Postgres version and **Type** to **DB Parameter Group**.
+
+1. Edit the new parameter group and set the `rds.logical_replication` parameter to `1`.
+
+1. [Associate the RDS parameter group to your database](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_WorkingWithDBInstanceParamGroups.html#USER_WorkingWithParamGroups.Associating).
+
+    Be sure to use the **Apply Immediately** option. The database must be rebooted in order for the parameter group association to take effect.
+
+    Do not move on to the next step until the database **Status** is **Available** in the RDS Console.
+
+1. Back in your `psql` shell, verify that replication is now enabled:
+
+    ``` sql
+    SELECT name,setting
+      FROM pg_settings
+      WHERE name = 'rds.logical_replication';
+    ```
+
+    ``` noftm
+            name             | setting
+    -------------------------+---------
+    rds.logical_replication  | on
+    (1 row)
+    ```
+
+    If replication is still not enabled, [reboot the database](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_RebootInstance.html).
+
+## Step 2. Create a publication
+
+Once logical replication is enabled, the next step is to create a publication with the tables that you want to replicate to Materialize. You'll also need create a user for Materialize and give the user permissions to manage replication.
+
+1. For each table that you want to replicate to Materialize, set the [replica identity](https://www.postgresql.org/docs/current/sql-altertable.html#SQL-ALTERTABLE-REPLICA-IDENTITY) to `FULL`:
+
+    ```sql
+    ALTER TABLE table1 REPLICA IDENTITY FULL;
+    ```
+
+    This setting makes sure sufficient information is written to the replication stream for  `UPDATE` and `DELETE` operations.
```

Comment:
Same here, not sure what sufficient information means and if there is specific data this requires or it refers to the amount of data.